csharp/test/Apache.Arrow.Tests/ArrowArrayConcatenatorTests.cs

```diff
@@ -495,6 +500,47 @@ public void Visit(UnionType type)
                         new[] { stringResultBuilder.Build().Data, intResultBuilder.Build().Data }));
             }
 
+            public void Visit(MapType type)
+            {
+                MapArray.Builder resultBuilder = new MapArray.Builder(type).Reserve(_baseDataTotalElementCount);
+                //Todo : Support various types
```

Comment:
what other types?

Reply:
This comment was copied from some other implementation; there are now three of these comments in this file. I don't think it's super important to test e.g. Map<foo> vs Map<bar> so I've deleted all three copies of this comment from ArrowArrayConcatenatorTests.cs and both copies from TestData.cs.